tensorflow/contrib/avro/ops/parse_avro_record.cc

```diff
@@ -0,0 +1,1630 @@
+#include <avro.h>
```

Comment:
Add copyright boilerplate (here and other files)

Reply:
Added boiler plates.